logging/Limiting_Access_to_Logs.md

```diff
@@ -56,60 +105,60 @@ be `mynamespace_mytenant1_admin`.
 
 Use the `/logging/bin/offboard.sh` script to remove logging access for a namespace or SAS Viya tenant. The script performs these actions: 
 
-- Removes the Kibana tenant space
-- Removes the access controls for the Kibana tenant space
+- Removes the Kibana-tenant space.
+- Removes the access controls for the Kibana-tenant space.
```

Comment:
I really dislike having periods at the end of bullets, especially when the text isn't a sentence.

Reply:
See above.

Reply:
The grammar check resolved the list items and the bullets are gone.